app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,16 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:id])
+
+    @years_of_experience = 0
```

Comment:
Really all you need in this controller action is line 3 - the rest of this logic should be encapsulated in a model method. 

You could create the following methods in your project model:
- contestant_count: The logic here would be `self.contestants.count`, it's a simple calculation but it's still a calculation so it should live at the model level per MVC conventions
- average_experience: You can use AR here! `self.contestants.average(:age)` and you can optionally add `.to_f.round(2)` to round it to 2 decimal spots

Remember that per MVC, we want slim controllers without much logic!